### PR TITLE
Fixes Issue #589 (Update prawn file)

### PR DESCRIPTION
**Pull request by shubhankardesh**
The attribute attending_social_events has been removed. The question "Are you attending any social events?" could now be asked by creating a custom question instead.

### CODE REVIEW

File: app/views/admin/registrations/index.pdf.prawn [hunk 1]
```diff
@@ -13,10 +12,6 @@ prawn_document(:force_download=>true, :filename => @pdf_filename) do |pdf|
     row << ""
     row << registration.name
     row << registration.email
-    if registration.attending_social_events
-      row << "X"
-    else
-      row << " "
     end
 
     if registration.attending_with_partner.to_s
```
Comment (ChrisBr) [inline app/views/admin/registrations/index.pdf.prawn]: I suppose we also moved attending_with_partner!
https://github.com/openSUSE/osem/blob/master/db/schema.rb#L310

**Comment by ChrisBr** [comment]
Hey @shubhankardesh,
thank you for your PR! :)

Have you tested your changes? Because I suppose we also moved the attribute attending_with_partner.
https://github.com/openSUSE/osem/blob/master/db/schema.rb#L310

Otherwise LGTM :+1:

**Comment by shubhankardesh** [comment]
Great! Well, it looks like the build has passed. We could move the attribute attending_with_partner as well. Maybe, it could be created as a custom question.

**Comment by ChrisBr** [comment]
Hey @shubhankardesh,
thanks for your response!

Yes, the build passed because there is no test for generating the .pdf file! Maybe you can investigate if there is a possibility to test it?

Attending with partner is already a dynamic question.
https://github.com/openSUSE/osem/blob/master/db/seeds.rb#L26

**Comment by shubhankardesh** [comment]
Yes of course!

**Comment by shubhankardesh** [comment]
Any pointers as to how to test the correct generation of the .pdf?

**Comment by ChrisBr** [comment]
@shubhankardesh testing the generation should be enough! Maybe a feature test!

**Comment by shayonj** [comment]
Great. While you are at it, I noticed few things while playing around with this bug.

You may want to take care of
```
    row << reg.attending_social_events
    row << reg.attending_with_partner
    row << reg.handicapped_access_required
```
from https://github.com/openSUSE/osem/blob/master/app/views/admin/registrations/index.xlsx.axlsx . As all of them are throwing errors. And finally setting the ```sheet.column_info``` reference to a not nil value, accordingly.

My two cents :)

Cheers

**Comment by differentreality** [comment]
@shubhankardesh Your changes do not work.
Please try the app with the changes you make before you send a PR!!
And as Chris said, add a relevant test. Where do you stand with that? Did you find how to test something about the feature of 'PDF generation' ?

**Comment by ChrisBr** [comment]
@shubhankardesh Any progress?

**Comment by ChrisBr** [comment]
No response since 4 weeks, so I will close this one!

**Pull request closed without merging**